.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: "daily"
```

Comment:
do we need it daily?

Reply:
Well, it doesn't cost anything and doesn't send you daily status updates or anything silly like that.

Running it daily spaces out updates so you get small chunks when they are available and you can take them when you have time instead of a wall of pull requests once a month. And if you want to deal with all your pull requests once a month, you still have the option to do so, just let them sit until you have time to deal with them.

You